src/references/templates.md

```diff
@@ -188,7 +188,7 @@ To learn more about importing and using **Community Content** templates, see the
 
 ## Nested Templates
 
-**Nested templates** is when a template contains children components that are also templates. This allows for modular workflow when making a system that has many different subcomponents. This also allows teams of creators to use asynchronous collaboration when changing one aspect of a big and complex design.
+**Nested templates** allows creators to place a template instance inside another template instance. This allows for modular workflow when making a system that has many different subcomponents. This also allows teams of creators to use asynchronous collaboration when changing one aspect of a big and complex design.
```

Comment:
So I would maybe think about the difference between an asset in project content which is not an instance, compared to a template added to the hierarchy which becomes an instance.

So maybe something more along the lines of this, though it's a bit more wordy:

**Nested Templates** allow creators to include template instances inside other templates. Nested templates keep a link to their own template asset, while being part of another template asset.